.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -58,7 +58,11 @@ will share this change with the people who need to know about it._
 
 - [ ] I've updated the [Developer Docs](https://developers.forem.com) or
       [Storybook](https://storybook.forem.com/) (for Crayons components)
-- [ ] This PR changes the Forem platform and our documentation needs to be updated. I have filled out the [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md) issue template so Community Success can help update the Admin Docs appropriately.
```

Comment:
This change probably shouldn't be part of this PR.

Reply:
Fixed, now the update is simply formatting issues that the commit hooks updated:)